### PR TITLE
Disabled failing booster in osio

### PR DESCRIPTION
**Pull request by corinnekrych**
as per https://github.com/openshiftio/openshift.io/issues/4547

### CODE REVIEW

**Comment by corinnekrych** [comment]
@gastaldi @Ladicek as discussed this one is failing too, let's disable it for now.

File: thorntail/community/health-check/booster.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ source:
 metadata:
   app:
     osio:
-      enabled: true
+      enabled: flase
```
Comment (Ladicek) [inline thorntail/community/health-check/booster.yaml]: flase -> false

Comment (Ladicek) [inline thorntail/community/health-check/booster.yaml]:
```suggestion
      enabled: false
```

**Comment by corinnekrych** [comment]
thanks @Ladicek really cool this suggestion review in gh
shame on me for such a typo :(

**Comment by Ladicek** [comment]
Agree, suggestions are great!

**Pull request merged**